main.go

```diff
@@ -78,21 +88,88 @@ func main() {
 		}
 	}
 
-	eventHandler, err := events.NewHandler(context.Background(), dbClient, azureClient, cfg.GCPProject, cfg.GCPRegion, cfg.GCPZone, cfg.AirflowChartVersion, cfg.JupyterChartVersion, cfg.DryRun, cfg.InCluster, log.WithField("subsystem", "events"))
+	eventHandler, err := events.NewHandler(
+		context.Background(),
+		dbClient,
+		azureClient,
+		cfg.GCP.Project,
+		cfg.GCP.Region,
+		cfg.GCP.Zone,
+		cfg.Helm.AirflowChartVersion,
+		cfg.Helm.JupyterChartVersion,
+		cfg.DryRun,
+		cfg.InCluster,
+		log.WithField("subsystem", "events"),
+	)
 	if err != nil {
 		log.WithError(err).Fatal("starting event watcher")
 		return
 	}
 	eventHandler.Run(10 * time.Second)
 
-	router, err := api.New(dbClient, azureClient, cfg.DryRun, cfg.SessionKey, cfg.AdminGroup, cfg.GCPProject, cfg.GCPZone, log.WithField("subsystem", "api"))
+	router := gin.New()
```

Comment:
Dette og ned til 169 trenger vel ikke `main.go` og forholde seg til? Dette vil jeg si er intern API-logikk, og burde vært pakket inn i noe.

Reply:
Enig! Tanken er å introdusere funksjonalitet for å initialisere hvert lag for seg, og det kan typisk trekkes ut av `main`, dette er fortsatt litt WIP, men skal få det inn i PR før merge 👌🏽